### PR TITLE
feat(invoice): add comments for documentation

### PR DESCRIPTION
**Pull request by devrrior**
## Describe your changes
I added openAPI comments for the invoice API endpoints

## Issue ticket number and link
Issue #110 

## Checklist before requesting a review
- [x] Create schema for Invoice Object
- [x] Add OpenAPI comments for endpoint

### CODE REVIEW

File: modules/invoice/api/invoice-routes.ts [hunk 1]
```diff
@@ -3,10 +3,92 @@ import * as core from 'express-serve-static-core';
 import { getQtyFromRequest } from '../../../utils/route-utils';
 import getRandomInvoices from '../utils/getRandomInvoices';
 
+/**
```
Comment (ageddesi) [inline modules/invoice/api/invoice-routes.ts:6]: Please could you move this definition to file of the object in 'consts' folder please :)

**Comment by ageddesi** [comment]
hi @devrrior Thank you for the PR, I have left one little comment and then its good to merge. 👍

File: modules/invoice/api/invoice-routes.ts (hunk 1 above)
Comment (devrrior) [reply modules/invoice/api/invoice-routes.ts]: Sure. It's done :D

**Comment by devrrior** [comment]
Sure. It's done :D

**Pull request merged**